pom.xml

```diff
@@ -130,27 +130,17 @@
         <plugins>
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.20.1</version>
```

Comment:
no need to set version?

Reply:
This version can be inherited from the parent pom file. No need to overwrite the version here.